dev-libs/skalibs/skalibs-2.8.1.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="General-purpose libraries from skarnet.org"
+HOMEPAGE="https://www.skarnet.org/software/skalibs/"
+SRC_URI="https://www.skarnet.org/software/${PN}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0/$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="doc ipv6 static-libs"
+
+RDEPEND=""
+DEPEND=""
+
+HTML_DOCS=( doc/ )
+
+src_prepare() {
+	default
+
+	# Avoid QA warning for LDFLAGS addition; avoid overriding -fstack-protector
+	sed -i -e 's/.*-Wl,--hash-style=both$/:/' -e '/-fno-stack-protector$/d' \
```

Comment:
This really looks like you are removing the whole line in two different ways, for no apparent reason.

Reply:
I already explained the reason why `/-Wl,--hash-style=both$/d` is not possible:

> In this case, it's the only line inside an if statement, so it's invalid syntax to simply remove the line.

Reply:
Hah, I'm sorry, this time I missed the `:` ;-).